engine/runtime/src/main/java/org/enso/interpreter/runtime/type/Types.java

```diff
@@ -3,7 +3,6 @@
 import com.oracle.truffle.api.dsl.TypeSystem;
 import com.oracle.truffle.api.interop.ArityException;
 import com.oracle.truffle.api.interop.UnsupportedTypeException;
-// import org.enso.interpreter.runtime.ConstantsGen;
```

Comment:
cleanup

Reply:
The `// import org.enso.interpreter.runtime.ConstantsGen;` is gone. Is that what you wanted, Dmitry?